lib/lib.go

```diff
@@ -683,8 +683,8 @@ type Instance struct {
 
 // Connect takes an instance online
 func (inst *Instance) Connect(ctx context.Context) (err error) {
-	oldRemoteClientExisted := inst.remoteClient != nil
-	log.Debugf("inst.Connect oldRemoteClientExisted=%t", oldRemoteClientExisted)
+	// oldRemoteClientExisted := inst.remoteClient != nil
+	// log.Debugf("inst.Connect oldRemoteClientExisted=%t", oldRemoteClientExisted)
```

Comment:
any reason why this is removed? Mind clarifying why we no longer need to track & decrement the releaser on `remoteClient`?

This might be why `TestConnect` is failing in CI

Reply:
ahh boo, I thought I was so cool and added a comment here ala uhLeeshUh, but it was a "pending" review, not a comment, here it is:

I am getting "negative wait group" errors in test-plans from the `oldRemoteClientExisted` 
```
  if oldRemoteClientExisted {
      inst.releasers.Done()
  }
```
I can't find a difference in the way we connect our normal qri instances than our test-plans qri instances, so I'm wondering if maybe this doesn't work exactly the way we expect? What was it added for?

Reply:
Interesting. My hunch is an interaction within the `cmd` package, probably along the `qri connect` codepath. I'm guessing there may be 2 invocations of `QriNode.GoOnline` when calling `qri connect`, and only 1 when using qri directly as a library.